plugins/logger/zap_adapter.go

```diff
@@ -17,20 +18,39 @@ func NewZapAdapter(zapLogger *zap.Logger) *ZapAdapter {
 	}
 }
 
+func separateFields(keyVals []interface{}) (fields []zap.Field, individualKeyVals []interface{}) {
```

Comment:
Please, do not use named return args :)

Reply:
Sorry, I'm new to Golang and wasn't aware of any issues with doing this? This should be resolved now.

Reply:
np :) I'll be glad to help you 👍🏻